aptos-move/framework/aptos-framework/sources/PropertyMap.move

```diff
@@ -106,6 +106,16 @@ module AptosFramework::PropertyMap {
         };
     }
 
+    public fun generate_string_vector(values: vector<vector<u8>>): vector<String> {
```

Comment:
Can this run into gas limit issues if the values vector is too large?

Reply:
yes, it could be running into limit like any other functions